src/components/MewTokenContainer/MewTokenContainer.vue

```diff
@@ -129,16 +126,29 @@ export default {
 
       return '52px';
     },
-  },
+    getIconSize() {
+      if (this.size.toLowerCase() === this.sizeOptions.small) {
+        return '24px';
+      }
+
+      if (this.size.toLowerCase() === this.sizeOptions.medium) {
+        return '32px';
+      }
+
+      return '35px';
```

Comment:
these numbers came from russ so need to confirm with him

Reply:
Yes. This design was confirmed by Russ.